client/settings/card-body/styles.scss

```diff
@@ -42,12 +42,6 @@
 		max-width: none;
 	}
 
-	// spacing in the "Express checkouts" settings page
-	.components-radio-control__option {
-		align-items: center;
-		display: flex;
-	}
-
```

Comment:
I had to remove this override because it was breaking the way the RadioControl was being displayed.